### DISCUSSION TITLE
Apply Settings / Changes To All Users Rather Than Only Current User

### DISCUSSION THREAD

**Discussion by bSmallPSSD** [valinet/ExplorerPatcher#1568] [Q&A]
Is there a way to Apply settings changes (particularly regarding the Start Menu and Taskbar layouts) to ALL users who sign into a particular computer, rather than just the Current User? As of right now, it appears that each new user who signs into my test computer has to manually import or adjust the settings .reg file the first time they sign in. Wondering if it's possible to push the Settings set by me (as Administrator) to affect everyone else's Start Menu and Taskbar when they sign into the machine, rather than having to redo everything for each new user account on the computer. Any help would be greatly appreciated. Thanks in advance.

**Comment by pyrates999** [comment]
You can do this through active directory's registry feature I think:

https://woshub.com/how-to-create-modify-and-delete-registry-keys-using-gpo/

**Reply by bSmallPSSD** [reply]
@pyrates999 Thank you, I'll take a look at this link, and see if it helps me out. We're actually in the process of transititioning away from Active Directory and moving over to Azure Active Directory and Microsoft Intune / Endpoint Manager for managing our users and computers moving forward. I'll continue to research this topic on my end as well, but if you have any other helpful links / ideas for me to look into in regards to setting up a Configuration Profile (or something else) within Intune Endpoint Manager to accomplish what I asked about in my previous post, that would be greatly appreciated. Thanks again for the reply.
